### ISSUE TITLE
nest-cli assets problem

### ISSUE THREAD

**Issue by jpx1024** [nestjs/nest-cli#1026]
## Current behavior
I configured assets according to the document, but nest-cli did not move YML to the dist folder.

## Input Code
`
{
    "collection": "@nestjs/schematics",
    "sourceRoot": "src",
    "assets": ["**/*.yml"]
}
`
![image](https://user-images.githubusercontent.com/66124276/106082554-dba3ed80-6155-11eb-95e3-fa7de4aae264.png)

**Comment by jpx1024** [comment]
> 请提供最低限度的复制品作为可复制的git repo

**Comment by jpx1024** [comment]
> Please provide a minimum reproduction as a clonable git repo

https://github.com/jpx1024/nest-template.git
thank you

**Comment by pabloiea1995** [comment]
Any updates on this? Thank you

**Comment by JanJakes** [comment]
I've just run into a (maybe) related issue – when running multi-arch Docker build (on GitHub Actions) I have the assets in the `dist` folder for `amd64` but **not** for `arm64`.

Are you @jpx1024 by any chance running on M1 or any kind of ARM? (Just tried your repo and it actually seems to work on amd64 – all the `.yml` files are copied to `dist/resources`).

**Comment by oliviermichaelis** [comment]
I can confirm that we apparently hit the same issue on `arm64`. The following `nest-cli.json` works for `amd64` but not for `arm64`:

```json
{
    "$schema": "https://json.schemastore.org/nest-cli",
    "collection": "@nestjs/schematics",
    "sourceRoot": "src",
    "compilerOptions": {
        "deleteOutDir": true,
        "assets": [
            "**/public/**/*"
        ],
        "watchAssets": true
    }
}
```

I verified this with the latest available release (`10.4.4`).

@kamilmysliwiec is there any chance to re-open this issue? Or should I create a new issue? :)

**Comment by slaveofcode** [comment]
Why was this issue closed when the problem is still happening?
